CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @clerkinc/backend-team
```

Comment:
@SokratisVidros should we give access to `@clerkinc/backend-team` to this repo or should i put each of us as codeowner. The current line raises the below error in github.
<img width="744" alt="Screenshot 2023-03-06 at 10 25 36" src="https://user-images.githubusercontent.com/3936408/223084208-18a62a35-12ad-4478-94e2-e585c1bdeaeb.png">